packages/block-template/src/base.module.scss

```diff
@@ -0,0 +1,251 @@
+  --font-size-base: clamp(1rem, 0.34vw + 0.91rem, 1.12rem);
+  --font-size-md: clamp(1.25rem, 0.61vw + 1.1rem, 1.58rem);
+  --font-size-lg: clamp(1.56rem, 1vw + 1.31rem, 2.11rem);
+  --font-size-xl: clamp(1.95rem, 1.56vw + 1.56rem, 2.81rem);
+  --font-size-xxl: clamp(2.44rem, 2.38vw + 1.85rem, 3.75rem);
+  --font-size-xxl: clamp(3.05rem, 3.54vw + 2.17rem, 5rem);
+
+  /* line heights */
+  --line-height-tight: 1.25;
+  --line-height-base: 1.5;
+  --line-height-loose: 1.75;
+
+  /* font family */
+  --font-family: system-ui, "Inter", "Open Sans", "Helvetica Neue", "Arial",
+  sans-serif;
+
+  /* border radius */
+  --border-radius-1: 0.25rem;
+  --border-radius-2: 0.5rem;
+  --border-radius-3: 1rem;
+
+  /* box shadow */
+  --shadow-1: 0 1px 3px 0 rgb(0 0 0 / 0.1), 0 1px 2px -1px rgb(0 0 0 / 0.1);
+  --shadow-2: 0 4px 6px -1px rgb(0 0 0 / 0.1), 0 2px 4px -2px rgb(0 0 0 / 0.1);
+  --shadow-3: 0 10px 15px -3px rgb(0 0 0 / 0.1), 0 4px 6px -4px rgb(0 0 0 / 0.1);
+  --shadow-4: 0 25px 50px -12px rgb(0 0 0 / 0.25);
+
+  /* border weight */
+  --border-weight-thin: 1px;
+  --border-weight-light: 2px;
+  --border-weight-base: 4px;
+
+  /* icon size */
+  --icon-size-base: 1rem;
+  --icon-size-large: 1.5rem;
+
+  /* –––––––––– Helper styles –––––––––– */
+
+  font-family: var(--font-family);
+  color: var(--primary-text-base);
+
+  /* Typography */
+
+  h1,
+  h2,
+  h3,
+  h4,
+  p,
+  li,
+  ol {
+    max-width: 60ch;
+  }
+
+  h1 {
+    font-size: var(--font-size-xxl);
+    margin: var(--space-4) 0 var(--space-2);
+    font-weight: 700;
+    color: var(--neutral-900);
+  }
+
+  h2 {
+    font-size: var(--font-size-xl);
+    margin: var(--space-4) 0 var(--space-2);
+    font-weight: 600;
+    color: var(--neutral-800);
+  }
+
+  h3 {
+    font-size: var(--font-size-lg);
+    margin: var(--space-4) 0 var(--space-2);
+    font-weight: 400;
+    color: var(--neutral-700);
+  }
+
+  h4 {
+    font-size: var(--font-size-md);
+    margin: var(--space-2) 0;
+    font-weight: 400;
+    color: var(--neutral-600);
+  }
+
+  h5 {
+    font-size: var(--font-size-base);
+    margin: var(--space-2) 0;
+    font-weight: 500;
+  }
+
+  p,
+  ol,
+  ul {
+    margin-bottom: var(--space-4);
+  }
+
+  p,
+  li {
+    font-size: var(--font-size-base);
+    line-height: var(--line-height-base);
+  }
+
+  p:global(.small-text) {
+    font-size: var(--font-size-xs);
+    color: var(--neutral-500);
+  }
+
+  hr {
+    border: 1px solid var(--border-color-light);
+  }
+
+  /* Buttons */
+
+  button {
+    background-color: var(--brand-primary-base);
+    color: var(--neutral-0);
+    border: none;
+    padding: var(--space-2) var(--space-4);
+    border-radius: var(--border-radius-1);
+    font-size: var(--font-size-sm);
+    line-height: var(--line-height-base);
+    transition: 250ms ease-in-out;
+  }
+
+  button:hover {
+    background-color: var(--brand-primary-hover);
+    cursor: pointer;
+  }
+
+  /* Box element for containers and sections */
+  :global(.box) {
```

Comment:
Not a blocker, just curious: can this still create clashes?

Reply:
So this will generate something like `.some-random-string .box` - if the EA styles `.box` globally, then yes, it will hit this, but this won't leak from the block to the EA.

Reply:
I thought `:global` would exclude `.some-random-string` from the output, but nevermind. Let’s see how it goes.

I’m also curious about the nature of `.some-random-string` in general. If it’s a hash just from the local relative file name, we may end up with duplicate random strings used in different blocks. I guess it’s more than that, so let’s try and see!

Reply:
Ah I see what you mean now. Without `:global` it’s `styles.box` but with global it’s `.some-random-string .box`. Got you!

Reply:
Basically just means `.box` works globally without using a dynamic classname, provided its within an element with the dynamic classname